python/test/unit/language/test_line_info.py

```diff
@@ -136,3 +155,6 @@ def test_line_info(func: str):
         assert (check_file_lines(file_lines, "test_line_info.py", 61))
         assert (check_file_lines(file_lines, "test_line_info.py", 62))
         assert (check_file_lines(file_lines, "test_line_info.py", 63))
+    elif func == "dot_combine":
+        assert (check_file_lines(file_lines, "test_line_info.py", 70))
```

Comment:
It's not obvious to me just from looking at the test why it's incorrect to include line 71, i.e. why we really don't want the line with the `+` to appear in the ptx.  Could we add a sentence or two explaining?

Reply:
Comments added